recipes-debian/libestr/libestr_debian.bb

```diff
@@ -0,0 +1,19 @@
+# base recipe: meta-oe/recipes-support/libestr/libestr_0.1.11.bb
+# base branch: warrior
+# base commit: 1a6f0053655db8657e4b6e710b16036397f1ad7a
+
+SUMMARY = "some essentials for string handling (and a bit more)"
+HOMEPAGE = "http://libestr.adiscon.com/"
+LICENSE = "LGPLv2.1"
+LIC_FILES_CHKSUM = "file://COPYING;md5=9d6c993486c18262afba4ca5bcb894d0"
+
+inherit debian-package
+require recipes-debian/sources/libestr.inc
+
+DEBIAN_UNPACK_DIR = "${WORKDIR}/${BPN}-${PV}"
+S = "${WORKDIR}/${BPN}-${PV}"
+
+do_debian_patch(){
```

Comment:
@AkitoTakada if libestr debian package doesn't have any patches, please set  ``` DEBIAN_QUILT_PATCHES = ""``` instead of override do_debian_patch(). Setting DEBIAN_QUILT_PATCHES is recommended. Because If this package will contain patches in the future, do_debian_patch() will makes error because DEBIAN_QUILT_PATCHES is empty but debian package have patches. So it's easy to handle debian patches.